modules/installation-osp-default-kuryr-deployment.adoc

```diff
@@ -0,0 +1,295 @@
+[id="installation-osp-default-kuryr-deployment"]
```

Comment:
Specify which assemblies include this module:
>// Module included in the following assemblies:
>//
>...

Reply:
Sorry, but I'm not sure what that means... This module is only referenced from the other module: modules/installation-osp-default-deployment.adoc. Do I need to include them there?

Reply:
ok, I'll copy the ones at modules/installation-osp-default-deployment.adoc. Though still... this is a module referenced in that module... did it this way to not add the content directly on modules/installation-osp-default-deployment.adoc. But if a module cannot be referenced by another module... maybe I need to move this there